dist/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <section class="main">
+      <div class="heading">
+        <h1>Leaderboard</h1>
+      </div>
+      <div class="leaderboard">
+        <div class="show-list">
+          <div class="show-list-head">
+            <h2>Recent scores</h2>
+            <button>Refresh</button>
+          </div>          
+          <ul id="leaderboard">
+            <li class="leaderboard-item" data-score="100">
+              <span class="name">Alice</span>
+              <span class="score">100</span>
+            </li>
+            <li class="leaderboard-item" data-score="90">
+              <span class="name">Bob</span>
+              <span class="score">90</span>
+            </li>
+            <li class="leaderboard-item" data-score="85">
+              <span class="name">Alex</span>
+              <span class="score">85</span>
+            </li>
+          </ul>
+        </div>  
+        <div class="add-item">
+          <div class="show-list-head">
+            <h2>Add your score</h2>
+          </div>
+          <form id="entry-form">
+            <input type="text" id="name" name="name" placeholder="Your name" required><br>
+            <input type="number" id="score" name="score" placeholder="Your score" required><br><br>
+            <input type="submit" value="Submit">
+          </form>
+        </div>
+      </div>   
+    </section>      
+  </body>
+</html>
```

Comment:
- [ ] Your [HTML](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), code should match the given wireframe, no styling is required at this stage